service.nextup.notification/service.py

```diff
@@ -34,16 +37,32 @@ def logMsg(self, msg, lvl=1):
     def ServiceEntryPoint(self):
         player = Player()
         monitor = xbmc.Monitor()
-
+        
         lastFile = None
         lastUnwatchedFile = None
+
+        ga = GoogleAnalytics()
```

Comment:
Yes but at very high level (i.e. As per google policy no user is actually identified). It serves two purposes. The first is to allow me to view any fatal exceptions and the second is to gauge the usage of the new postplay screen compared to the older one. Again as per google policy users can opt out by setting it in the settings.

If this is against Kodi policy please let me know and I'll remove it.